File: soutLink-backend/src/main/java/com/soutLink/supervisor/dto/AppointmentDetailsResponse.java
package com.soutLink.supervisor.dto;

import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class AppointmentDetailsResponse {
    private StudentAppointmentResponse studentAppointment;
    private String projectTitle;
    private List<StudentResponse> students;
}

File: soutLink-frontend/src/app/services/models/appointment-details-response.ts
/* tslint:disable */
/* eslint-disable */
import { StudentAppointmentResponse } from '../models/student-appointment-response';
import { StudentResponse } from '../models/student-response';
export interface AppointmentDetailsResponse {
  projectTitle?: string;
  studentAppointment?: StudentAppointmentResponse;
  students?: Array<StudentResponse>;
}
